data/human/spaceport conversations.txt

```diff
@@ -66,40 +31,56 @@ mission "ZZCU Sad Gamblers -S"
 		random < 1
 	on offer
 		conversation
-			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success, and your attention is drawn as one of them wins a moderate sum in a 12th division win. It's a day or two worth of standard pay.`
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success, and your attention is drawn as one of them wins a moderate sum in a 12th division win. It's one or two day's worth of standard pay.`
 			`	It's difficult not to pity them as they immediately buy more tickets with the whole value.`
 				decline
 
 
-#               Republic Only               #
 
-mission "ZZCU Gift Store Interaction"
+mission "Gift Store Interaction"
 	minor
 	source
 		government "Republic"
-		attributes "deep"
+		attributes "tourism"
 		not attributes "station"
 	to offer
 		random < 1
 	on offer
 		conversation
-			`As you wander the spaceport with a snack, waiting for your ship to be refueled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylized representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, not too small for your fingers, and the lip of it looks like it would be comfortable to drink out of.`
-			`	Two young adults wander into the aisle just a couple of meters away. They're clearly from abroad judging by their conversation occurring in some local dialect which is neither local nor one you understand. They're cooing over an item they've found, but when one flips it over to look at the price tag, there is an immediate exchange of looks and expressions that does not require translation. The item ends up back on the shelf with a giggle and a head-shake.`
```

Comment:
This pertains to this: `* Changed the talk of dialects to accents. Dialects are language variants; even if they were speaking a different dialect of the same language you're speaking, you should be able to understand them. If they're speaking a different language, then talking about the dialect doesn't matter, and talking about the different language would make more sense. But the distinction between accents in different region of space has been used before in other missions, hence the switch to highlighting the accents.`

The entire point of this bit was to capture that you explicitly did not understand any of the spoken words, but nevertheless had a clear idea of exactly what was going on. Going with accents I feel takes that the opposite direction and is incongruous with the following sentence of relying on looks and expressions.

The entry for dialects also explicitly includes the comment that, "Some dialects of a language are not mutually intelligible in spoken form."